applications/accounts/deploy/templates/api-user-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
This is fine but why not use the harness/secret? Shouldn't it be the default way to create secrets?

Reply:
@filippomc I think we can't set a random value with the default CH secrets , only a fixed string. One of our goals is to not store passwords in git so a fixed string isn't the best.

Reply:
note: probably we should allow to create random passwords in the default CH secrets

Reply:
@filippomc yeah I was thinking that too :-)

Reply:
https://github.com/MetaCell/cloud-harness/issues/387

Reply:
implemented random secret values and use CH secret managment here: https://github.com/MetaCell/cloud-harness/pull/389